nautobot/extras/choices.py

```diff
@@ -482,14 +482,19 @@ class SecretsGroupAccessTypeChoices(ChoiceSet):
     )
 
 
+
 class SecretsGroupSecretTypeChoices(ChoiceSet):
     TYPE_KEY = "key"
     TYPE_PASSWORD = "password"  # noqa: S105  # hardcoded-password-string -- false positive
     TYPE_SECRET = "secret"  # noqa: S105  # hardcoded-password-string -- false positive
     TYPE_TOKEN = "token"  # noqa: S105  # hardcoded-password-string -- false positive
+    TYPE_PRIVKEY = "privkey"  # noqa: S105  # hardcoded-password-string -- false positive
+    TYPE_AUTHKEY = "authkey"  # noqa: S105  # hardcoded-password-string -- false positive
```

Comment:
We have up to 32 characters to use for these values without requiring a database schema migration, so why not `private-key`, `authentication-key`, etc.?